Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -607,13 +612,7 @@ extension NSManagedObjectContext: MessageDatabaseSession {
             }
         )
 
-        // If a user is able to mention someone,
-        // most probably we have that user already saved in DB.
-        // Ideally, this should be `loadOrCreate` but then
-        // we miss non-optional fields of DTO and fail to save.
-        message.mentionedUsers = Set(
```

Comment:
What's not clear to me is: when would the `mentionedUsers` value be set, now that we delete this code? We mention in a comment above that this should still be used for reading mentioned users.

Reply:
This is set on the `saveMessage()` whenever we fetch channels, messages from the servers